core/services/blockhashstore/common.go

```diff
@@ -134,3 +139,15 @@ func SendingKeys(fromAddresses []ethkey.EIP55Address) []common.Address {
 	}
 	return keys
 }
+
+func RegisterLogPoller(coordinator Coordinator, logPoller logpoller.LogPoller, filterName string) error {
```

Comment:
I think it would make more sense API-wise to pass in the coordinator addresses here rather than using `coordinator.Addresses` which feels clunky.